src/Shared/SFA.DAS.SharedOuterApi/InnerApi/Requests/EmployerAccounts/GetUserByEmailRequest.cs

```diff
@@ -0,0 +1,15 @@
+﻿using SFA.DAS.SharedOuterApi.Interfaces;
+
+namespace SFA.DAS.SharedOuterApi.InnerApi.Requests.EmployerAccounts;
+
+public class GetUserByEmailRequest : IGetApiRequest
+{
+    public string Email { get; }
+
+    public GetUserByEmailRequest(string email)
+    {
+        Email = email;
+    }
+
+    public string GetUrl => $"api/user?email={Email}";
```

Comment:
Set as const

Reply:
Can't be a constant, it's dynamically built